README.md

```diff
@@ -1 +1,180 @@
-# Hello-Microverse-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="ghaus-logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Please either remove this logo or correct the path image because as I can see that your image link is broken which makes your `README` file looks unprofessional. Kindly try to focus to make your documentation error-free and be more professional.